Add solution of chatglm2-6b cannot set attribute to Readme (#27)
openvinotoolkit/openvino.genai: llm_bench/python/README.md

```bash
python ./benchmark.py -m models/llama-2-7b-chat/pytorch -d CPU --torch_compile_backend openvino
```

> 

Reviewer: ```suggestion
# Notes
```

> ## Fix tokenizer initialization issue AttributeError of chatglm2-6b: can't set attribute

Reviewer: ```suggestion
## chatglm2-6b - AttributeError: can't set attribute
```

When running the chatglm2-6b downloaded from hugginface and converted to openvino IR, the following error may occur：
```bash
AttributeError: can't set attribute

> ```

Reviewer: Reproduced with https://huggingface.co/THUDM/chatglm2-6b 7fabe56db91e085c9c027f56f1c654d137bdba40

Author: updated

> The solution is to modify `tokenization_chatglm.py` in the openvino IR directory and insert the following code in `line 73`

Reviewer: ```suggestion
As on https://huggingface.co/THUDM/chatglm2-6b/discussions/99<br>
Solution: update `tokenization_chatglm.py` as following:
```

```bash
python ./benchmark.py -m models/llama-2-7b-chat/pytorch -d CPU --torch_compile_backend openvino
```
# Notes
## chatglm2-6b - AttributeError: can't set attribute

> When running the chatglm2-6b downloaded from hugginface and converted to openvino IR, the following error may occur：

Reviewer: ```suggestion
Download chatglm2-6b from hugginface, convert to OpenVINO IR files and run with benchmark.py, the following error may occur：
```

```bash
AttributeError: can't set attribute
```
Reproduced with https://huggingface.co/THUDM/chatglm2-6b 7fabe56db91e085c9c027f56f1c654d137bdba40 <br />
As on https://huggingface.co/THUDM/chatglm2-6b/discussions/99 <br />
Solution: update `tokenization_chatglm.py` as following: <br />
```Python
          self.vocab_file = vocab_file
          self.tokenizer = SPTokenizer(vocab_file)
 +        kwargs.pop("eos_token", None)
 +        kwargs.pop("pad_token", None)
 +        kwargs.pop("unk_token", None)
          self.special_tokens = {
              "<bos>": self.tokenizer.bos_id,

>               "<eos>": self.tokenizer.eos_id,

Reviewer: ```suggestion
              "<eos>": self.tokenizer.eos_id,
              
> The solution works for chatglm3-6b as well.
```

```bash
python ./benchmark.py -m models/llama-2-7b-chat/pytorch -d CPU --torch_compile_backend openvino
```
# Notes
## chatglm2-6b - AttributeError: can't set attribute
When running the chatglm2-6b downloaded from hugginface and converted to openvino IR, the following error may occur：
```bash
AttributeError: can't set attribute
```
Reproduced with https://huggingface.co/THUDM/chatglm2-6b 7fabe56db91e085c9c027f56f1c654d137bdba40 <br />
As on https://huggingface.co/THUDM/chatglm2-6b/discussions/99 <br />
Solution: update `tokenization_chatglm.py` as following: <br />

> ```Python
> {
>           self.vocab_file = vocab_file
>           self.tokenizer = SPTokenizer(vocab_file)
>  +        kwargs.pop("eos_token", None)
>  +        kwargs.pop("pad_token", None)
>  +        kwargs.pop("unk_token", None)
>           self.special_tokens = {
>               "<bos>": self.tokenizer.bos_id,
>               "<eos>": self.tokenizer.eos_id,
> }

Reviewer: ```suggestion
```Python
          self.vocab_file = vocab_file
          self.tokenizer = SPTokenizer(vocab_file)
 +        kwargs.pop("eos_token", None)
 +        kwargs.pop("pad_token", None)
 +        kwargs.pop("unk_token", None)
          self.special_tokens = {
              "<bos>": self.tokenizer.bos_id,
              "<eos>": self.tokenizer.eos_id,
```

> > The solution works for chatglm3-6b as well.

Reviewer: ```suggestion

> The solution works for chatglm3-6b as well.
```
